protopipe_grid_interface/scripts/submit_jobs.py

```diff
@@ -417,6 +417,34 @@ def main():
     log.debug("Particle type: %s", particle)
     log.debug("Energy estimation: %s", estimate_energy)
 
+    # Upload analysis configuration file for provenance
+    if switches["upload_analysis_cfg"]:
```

Comment:
as long as the user knows what dry means, I don't think it's any different. I just have trouble reading long bits of code with nested IFs, since it's hard to follow.   It would be better to just have an info at the top saying 
```py3
if switches['dry']:
   log.info("Dry run enabled. No files will be uploaded")
```

Reply:
OK this might be better